.github/policies/eventResponderTask.GithubActionPROpened.yml

```diff
@@ -0,0 +1,38 @@
+id: eventResponderTask.GithubActionPROpened
```

Comment:
It would be nice to add some comments in here describing why it is doing what it is doing.

Reply:
Done